src/diabatic/wide_band_bath_discretisation.jl

```diff
@@ -8,19 +8,19 @@ function fillbathstates!(out::Hermitian, bath::WideBandBathDiscretisation)
     copy!(diagonal, bath.bathstates)
 end
 
-function fillbathcoupling!(out::Hermitian, coupling::Real, bath::WideBandBathDiscretisation)
+function fillbathcoupling!(out::Hermitian, coupling::Real, bath::WideBandBathDiscretisation, coupling_rescale)
```

Comment:
do you want to have coupling_rescale in `fillbathcoupling!` and `setcoupling!` as an optional keyword? there might be some use cases where these functions might be needed without. 

also, we will probably at some point need multiple despatch for these as we will have scenarios where the coupling isn't a global real constant. Therefore, I recommend specifically setting the type to Real already here and in the other similar functions.